src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/proxy/ProxyControl.java

```diff
@@ -161,6 +171,10 @@
     private static final String SAMPLER_TYPE_HTTP_SAMPLER_HC3_1 = "1";
     private static final String SAMPLER_TYPE_HTTP_SAMPLER_HC4 = "2";
 
+    private static final int HTTP_SAMPLER_NUMBERING_MODE_PREFIX = 0;
+    private static final int HTTP_SAMPLER_NUMBERING_MODE_SUFFIX = 1;
+    private static final int HTTP_SAMPLER_NUMBERING_MODE_NO_NUMBER = 2;
```

Comment:
Could the information encoded here, in other constants in this class and also other classes be part of an Enum?

Reply:
All modifications are done, what's next  ?
Did this pull request will merge ?
If yes, i will modify the documentation for the "Http(s) Test Script Recorder"